x-pack/plugins/observability_solution/infra/public/plugin.ts

```diff
@@ -166,7 +166,7 @@ export class Plugin implements InfraClientPluginClass {
                       label: 'Infrastructure',
                       sortKey: 300,
                       entries: [
-                        { label: 'Inventory', app: 'metrics', path: '/inventory' },
+                        { label: 'Infrastructure Inventory', app: 'metrics', path: '/inventory' },
```

Comment:
This file doesn't seem to use translations, this is okay?

Reply:
We should translate them. You can either open an issue or do it now. It's up to you.

Reply:
I decided to update just the Infra labels to be translated. There's others for I think logs that also are not translated, but I'll have that be for a separate issue.